roles/init/tasks/postgres.yml

```
@@ -247,16 +247,37 @@
   with_dict: >-
     {{ postgres_tablespaces|default({}) }}
 
+# Determine the upstream primary for this host, that is, the primary at
+# the root of the cascading replication setup. This may be the instance
+# itself, if it's a primary.
+
+- name: Record name of the upstream primary
+  set_fact:
+    upstream_primary: "{{ inventory_hostname|upstream_root(hostvars) }}"
+  when: >
+    'postgres' in role
+
+- block:
+  - name: Prevent repmgr is installed on wrong BRD nodes when creating a replica
```

```suggestion
  - name: Allow physical replicas of BDR subscriber-only nodes
```
(Since this condition is in itself not about repmgr, but about physical streaming replication)

---

Mario, I see you have given the suggestion a thumbs-up but I also see it's not applied. Looks like you have missed it.

---

I was waiting for the confirmation message in the assert before to send this as well